backend/src/test/resources/testResultUpload/test-results-upload-valid-as-fhir.json

```diff
@@ -116,7 +116,7 @@
         "subject": {
           "reference": "Patient/1234"
         },
-        "effectiveDateTime": "2021-12-20T00:00:00Z",
+        "effectiveDateTime": "2021-12-20T14:00:00.000Z",
```

Comment:
reflects the date in the CSV since we don't round to start of day in bulk upload code anymore